.expeditor/create-manifest.rb

```diff
@@ -88,6 +103,15 @@ def get_latest(channel, origin, name)
   end
 end
 
+def purge_fastly_path(host, url)
```

Comment:
Do we know for sure that Instant Purging is enabled for our fastly token? (I assume it is, but I've never had occasion to try so I don't know for sure.)